CONTRIBUTING.md

```diff
@@ -361,11 +361,21 @@ There are a few other build options available:
 
 ### Deprecation policy
 
-Qiskit Experiments is part of Qiskit and, therefore, the [Qiskit Deprecation
-Policy](https://github.com/Qiskit/qiskit/blob/main/DEPRECATION.md) fully applies here.
-Public-facing changes must come with a deprecation warning for at least three months or
-two version cycles before the old feature is removed. Deprecations can only happen on
-minor releases and not on patch releases.
+Qiskit Experiments's deprecation policy is based on Qiskit's [Deprecation
+Policy](https://github.com/Qiskit/qiskit/blob/45e42525224e9e1857300e7e5529273fe619c9e4/DEPRECATION.md)
+prior to its 1.0 release. Public-facing changes must come with a deprecation warning
+for at least three months or two version cycles before the old feature is removed.
+Deprecations can only happen on minor releases and not on patch releases.
+
+The timeline for deprecating an existing feature is as follows:
+
+* Minor release 1: An alternative path is provided. A `PendingDeprecationWarning` should be issued
+  when the old path is used, indicating to users how to switch to the new path and the release
+  in which the old path will no longer be available.
+* Minor release 2: The `PendingDeprecationWarning` becomes a `DeprecationWarning`. The release note
+  should indicate the feature has been deprecated and how to switch to the new path.
+* Minor release 3: The old feature is removed. The release note should indicate that the feature has
+  been removed and how to switch to the new path.
```

Comment:
@wshanks I'm linking to the commit where `DEPRECATION.md` was added to Qiskit. Not sure if there's a better link.

What do you think about this timeline? It's faster than the Qiskit policy, which says the old feature cannot be removed in the minor release immediately after the warnings, but that would be far too long for us. The other thing is we don't use `PendingDeprecationWarning` that much right now, and it creates more code churn than just starting with a `DeprecationWarning` and keeping that for two cycles.

Reply:
Thanks, updated the link. I broke the changes into three types in 553f502.